app/assets/javascripts/templates/shop_variant.html.haml

```diff
@@ -6,20 +6,16 @@
       %i.ofn-i_056-bulk><
       %em><
         \ {{::'bulk' | t}}
+  .small-3.medium-3.large-3.columns.variant-price
+    -# Now in a template in app/assets/javascripts/templates !
```

Comment:
I think this comment can be deleted.
templates in app/assets/javascripts/templates are not a new thing now.